src/include/cache.i

```diff
@@ -168,14 +169,29 @@ __wt_cache_dirty_leaf_inuse(WT_CACHE *cache)
     return (__wt_cache_bytes_plus_overhead(cache, cache->bytes_dirty_leaf));
 }
 
+/*
+ * __wt_cache_bytes_updates --
+ *     Return the number of bytes in use for updates.
+ */
+static inline uint64_t
+__wt_cache_bytes_updates(WT_CACHE *cache)
+{
+    uint64_t bytes_updates;
+
+    bytes_updates = __wt_safe_sub(
```

Comment:
This probably deserves a comment - describing that it's racey so possible that it's possible to go negative.